src/snap.c

```diff
@@ -102,7 +102,8 @@ _snap_next_edge(struct view *view, int start_pos, const struct snap_search def,
 	struct view *v;
 	int p = max;
 	for_each_view(v, &server->views, LAB_VIEW_CRITERIA_CURRENT_WORKSPACE) {
-		if (v == view || v->output != output || v->minimized || v->maximized) {
+		if (v == view || v->output != output || v->minimized
+				|| view->maximized != VIEW_AXIS_NONE) {
```

Comment:
I am not sure if we want to ignore windows as snap target to that are not fully maximized.
Maybe here a `view->maximized == VIEW_AXIS_BOTH` is more intuitive for a user wanting to align the windows with each other.

Reply:
Sounds good, I'm not really familiar with the snap code.

Reply:
Oh, also just detected by testing: it should be `v->maximized` rather than `view->maximized`.

Reply:
Oops!

Reply:
Pushed a [fixup] commit for this.